src/features/auth/frames/component-test-frame.tsx

```
@@ -1,3 +1,6 @@
+import { Link } from '@tanstack/react-router';
```

ESLint

[error] 'Link' is defined but never used.

(@typescript-eslint/no-unused-vars)